packages/backend/server/schema.prisma

```diff
@@ -475,6 +476,36 @@ model AiWorkspaceEmbedding {
   @@map("ai_workspace_embeddings")
 }
 
+enum AiJobStatus {
+  pending
+  running
+  finished
+  claimed
+  failed
+}
+
+model AiJobs {
+  id          String      @id @default(uuid()) @db.VarChar
+  workspaceId String      @map("workspace_id") @db.VarChar
+  blobId      String      @map("blob_id") @db.VarChar
+  createdBy   String?     @map("created_by") @db.VarChar
+  // job type, like "transcription"
+  type        String      @db.VarChar
```

Comment:
Is there any special consideration for not choosing database enum? Is it to facilitate quick addition of types without needing database changes in the future?